guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -189,6 +189,13 @@ gpgcheck=1
 ----
 endif::[]
 
+. Add `tomcat` and `tomcat-lib` to the `/etc/leapp/transaction/to_remove` file:
```

Comment:
From a wording perspective, I am not sure whether this should read "Add ..." as it's literally repeating the commands in text form, or do "Force the removal of Tomcat by adding …" or "Help dnf to find the right packages to upgrade by adding …"

But the command is right and will make the upgrade work again, so I am going to ACK this one regardless of the wording.